app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
 end
```

Comment:
The comment model should have a method that updates the comments counter for a post. Kindly add this method and submit for a re-review.

Reply:
Changes requested have been made ✅ Thank you @muneebulrehman